smartcar/vehicle.py

```diff
@@ -44,6 +46,9 @@ def __init__(self, vehicle_id: str, access_token: str, options: dict = None):
             if options.get("version"):
                 self._api_version = options["version"]
 
+            if options.get("flags"):
+                self._flags = options["flags"]
```

Comment:
Need to update the code to send the flags as query parameters

Make sure to serialize the flags into a string before you send them (search around in the code base to see how this can be achieved)